### ISSUE TITLE
Can't see list of /homes

### ISSUE THREAD

**Issue by erbara** [Xquiset/SetHomes#6]
### I can set up multiple homes and access to them as I should.

When I do /homes, the list doesn't show, although there are **multiple saves on the file??**

I know almost when this problem started, though I don't know how to fix it.
I have a server in Aternos. I've been using this plugin with 0 problems for a day, until I wanted a friend to join.
My friend doesn't have a Minecraft premium account, so I activated an option to "crack" de server, so that he gets to join.

After that, the list didn't work. No error showed up on console.
![2020-07-22 19_18_12-Minecraft 1 16 1 - Multijugador (servidor de terceros)](https://user-images.githubusercontent.com/26657017/88234892-23e6ce00-cc50-11ea-9f76-83b5fa62c362.jpg)

In between I installed the ProtectionStone plugin, which required to install WorldEdit and WorldGuard. If the command /homes stopped working after this, I've got no clue.

I've tried deleting the plugin/sethome folder, and start from sratch (server is new so I didn't care) and same still happened.

I've also tried to copy the previous saved home coords (I had a backup in a file), and deleting them too. I still can access those homes, altough homes.yml looks empty.

### Second bug

Just now I've tried to add multiple homes to test something else before posting this issue, and I've found out that I can set more homes. The current limit (even before screenshot) was set to 20, and I've only used up 3.
_I've already read in another issue that you recommended not to use LuckyPerms but another permission plugin, but this worked fine with LuckyPerms yesterday, right until the switch I've made._

![2020-07-22 19_23_27-Minecraft 1 16 1 - Multijugador (servidor de terceros)](https://user-images.githubusercontent.com/26657017/88235260-df0f6700-cc50-11ea-98df-c9fd71f7a53f.jpg)

Thanks in advance for your time.

**Comment by erbara** [comment]
So, although I tried to read everything prior my post (don't like wasting anyones time), I didn't read on the github page about the command "/setmax (group) (number)

Before the problem I had, modifying the permission trough LuckyPerms worked just fine. After posting this, I couldn't change max homes or see them listed (other than minecraft chat auto complete).
I've just tried it, and wonders! Now the max is properly set.

Now, also, homes are showing up properly. 
![2020-07-22 19_23_27-Minecraft 1 16 1 - Multijugador (servidor de terceros)](https://user-images.githubusercontent.com/26657017/88452160-c363b680-ce32-11ea-8bbd-82515a7acda4.jpg)

I don't know what caused this, and why this "fixed" it.
This erased the previos saved /homes, but I'm still happy with turnaround.

Hope this helps with someone's else problem.

**Comment by erbara** [comment]
After server reboot, no longer can add new homes.
/homes shows properly the list of current homes, but can't add more, not even as OP.

**Comment by erbara** [comment]
Found the cause.
There was an incompatibility with the puglin Ultimate_Economy.
Although this pluggin was the only one I was using for /homes, Ultimate_Economy brought support for homes too.

When I realized that I had homes not listed in /homes, I understood the problem was something else. I disabled this pluggin, and still could use homes.
Finally I found out this was the problem. Could disable the homes feature in the economy plugin, and everything works as expected.
